ucsb-cs156-w24/team03-w24-5pm-3
# Created CreatePage for Articles

Implemented CreatePage for Articles
Steps for testing:
1. Log in
2. Navigate to the articles create page
3. Enter the desired data and press submit
4. Go to swagger-ui and check if data is entered correctly in the database

<img width="1440" alt="Screenshot 2024-02-15 at 6 23 16 PM" src="https://github.com/ucsb-cs156-w24/team03-w24-5pm-3/assets/79371675/f36ed381-ca2a-4369-9520-2f84a2752680">
